libdeno/deno.h

```diff
@@ -76,7 +76,7 @@ int deno_execute(Deno* d, void* user_data, const char* js_filename,
 // Return value: 0 = fail, 1 = success
 // Get error text with deno_last_exception().
 int deno_execute_mod(Deno* d, void* user_data, const char* js_filename,
-                     const char* js_source);
+                     const char* js_source, int is_prefetch);
```

Comment:
Same comment as above regarding the name of this argument

Reply:
I am making them all bools now instead (C++, extern C and Rust)